src/Options/index.js

```diff
@@ -221,10 +221,49 @@ export interface ParseServerOptions {
   playgroundPath: ?string;
   /* Callback when server has started */
   serverStartComplete: ?(error: ?Error) => void;
+  /* Callback when server has started and before running schemas migration operations if schemas key provided */
+  beforeSchemasMigration: ?() => void | Promise<void>;
+  /* Rest representation on Parse.Schema https://docs.parseplatform.org/rest/guide/#adding-a-schema */
+  schemas: ?(JSONSchema[]);
   /* Callback when server has closed */
   serverCloseComplete: ?() => void;
 }
 
+export interface FieldType {
+  type:
+    | 'String'
+    | 'Boolean'
+    | 'File'
+    | 'Number'
+    | 'Relation'
+    | 'Pointer'
+    | 'Date'
+    | 'GeoPoint'
+    | 'Polygon'
+    | 'Array'
+    | 'Object'
+    | string;
```

Comment:
Why the last `| string;`?

Reply:
Because one day i have seen somewhere in the docs that Parse Server support custom data types (like: "Address").
May be it was removed.

Reply:
Do you have a reference for this?